listenbrainz/db/stats.py

```diff
@@ -120,62 +116,19 @@ def get_user_stats(user_id, columns):
         return dict(row) if row else None
 
 
-def get_user_artists(user_id):
+def get_user_artists(user_id: int) -> Optional[UserArtistStatRow]:
     """Get top artists for user with given ID.
 
         Args:
             user_id (int): the row ID of the user in the DB
 
         Returns:
-            A dict of the following format
-            {
-                'user_id' (int): the row ID of the user in the DB,
-                'artist'  (dict): artist stats for the user, see below for better description
-                'last_updated' (datetime): datetime object representing when
-                                        this stat was last updated
-            }
-
-
-            the `artist` dict will be of the following format:
-            {
-                'all_time': all time artist listen counts for the user
-                            calculated by listenbrainz.stats.user.get_top_artists
-                'count': the total number of artists this user has listened to
-                                calculated by listenbrainz.stats.user.get_artist_count
-            }
-
-            In general, the `artist` dict will contain all artist related stats
-            calculated for the user in listenbrainz.stats.user, keyed by stat name.
+            UserArtistStatRow: a row of statistics.user but only with artist json, user id and last_updated
     """
-    return get_user_stats(user_id, 'artist')
-
-
-def get_all_user_stats(user_id):
```

Comment:
Removed this function because it was dead code not used anywhere, we can bring it back if needed later.